tornjak-frontend/src/components/tornjak-api-helpers.js

```diff
@@ -252,6 +254,44 @@ class TornjakApi extends Component {
       })
   }
 
+  // clusterDelete - 
+  clusterDelete = (serverName, inputData, clustersListUpdateFunc, globalClustersList) => {
+    axios.post(GetApiServerUri("/manager-api/tornjak/clusters/delete/") + serverName, inputData,
+      {
+        crossdomain: true,
+      })
+      .then(response => {
+        clustersListUpdateFunc(globalClustersList.filter(el =>
```

Comment:
you probably want to pass in a closure to do what you want instead of putting the actual code logic in here. other thing you can do is to make this function use await to make sure it finishes and return an error code, the caller of `clusterDelete` can then determine what to do - that way you don't need to pass in `clustersListUpdateFunc` or `globalClustersList`